src/webgpu/wgsl_language_feature.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * @fileoverview Declares a type for each string tested against
+ * the WGSLLanguageFeatures property of the GPU object.
+ */
```

Comment:
Unfortunately I don't think TSDoc supports file-level documentation. This is getting attached to `kKnownWGSLLanguageFeatures` (visible in vscode or `docs/tsdoc`).

Looking into this it seems like TSDoc is kind of dying and we should probably use JSDoc? I'll attempt this sometime.